wrappers/kn/build.gradle.kts

```diff
@@ -0,0 +1,213 @@
+import io.github.isning.gradle.plugins.cmake.params.CustomCMakeParams
+import io.github.isning.gradle.plugins.cmake.params.entries.CustomCMakeCacheEntries
+import io.github.isning.gradle.plugins.cmake.params.entries.asCMakeParams
+import io.github.isning.gradle.plugins.cmake.params.entries.platform.ModifiablePlatformEntriesImpl
+import io.github.isning.gradle.plugins.cmake.params.entries.plus
+import io.github.isning.gradle.plugins.cmake.params.plus
+import io.github.isning.gradle.plugins.kn.krossCompile.invoke
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
+import java.util.*
+
+plugins {
+    alias(libs.plugins.kotlinMultiplatform)
+    alias(libs.plugins.krossCompile)
+    `maven-publish`
+    signing
+}
+
+group = "io.github.zxing-cpp"
+version = "2.2.1"
+
+Properties().apply {
+    rootProject.file("local.properties").takeIf { it.exists() && it.isFile }?.let { load(it.reader()) }
```

Comment:
This block looks weird to me. There is no `local.properties` in your patch and we don't have this manual parsing in the Android build.gradle.kts file either.

Reply:
It's for loading some local needed properties for gradle plugin.
It's not used in ci now but it might be used for setting crosscompiling toolchain and executable path when you dont want to add an global environment variable (espically when developing locally, you might dont want to set default toolchain).

Reply:
It's for loading some local needed properties for gradle plugin.
It's not used in ci now but it might be used for setting crosscompiling toolchain and executable path when you dont want to add an global environment variable (espically when developing locally, you might dont want to set default toolchain).

Reply:
The reason why andorid don't need to do this in build.gradle.kts is android has it plugin parsing these.
But I didn't put related logic in my plugin because it is not a "standard"(official behavoiur from gradle) reading properties from `local.properties`

Reply:
Well, if the Android plugin thought this should be a standard behavior, I'd argue that is the right move. But you know *a lot* more about this gradle business than I do, so I'm fine with that. Thanks for explaining.